build.xml

```diff
@@ -78,6 +78,9 @@
             <arg value="1s/5\.0\.1/5.0.2/;18s/5:1:0/5:2:0/"/>
             <arg value="${workdir}/cpp/log4tango/configure.in"/>
         </exec>
+        <exec executable="/bin/bash" dir="${workdir}/cpp/log4tango" failonerror="true" failifexecutionfails="true">
+            <arg value="${basedir}/bootstrap"/>
+        </exec>
         <exec executable="autoconf" dir="${workdir}/cpp/log4tango" failonerror="true" failifexecutionfails="true">
```

Comment:
`bootstrap` will run autoconf as a last step. Consider removing this line.

Reply:
I would do it other way around - remove autoconf from bootstrap

Reply:
Good catch! I removed it in commit b0d9aa9877680fe26ac86ea985f378e240009faf.